crates/bitwarden-wasm-internal/src/crypto.rs

```diff
@@ -0,0 +1,37 @@
+use std::rc::Rc;
+
+use bitwarden::{
+    mobile::crypto::{InitOrgCryptoRequest, InitUserCryptoRequest},
+    Client,
+};
+use wasm_bindgen::prelude::*;
+
+use crate::error::Result;
+
+#[wasm_bindgen]
+pub struct ClientCrypto(pub(crate) Rc<Client>);
+
+#[wasm_bindgen]
+impl ClientCrypto {
+    /// Initialization method for the user crypto. Needs to be called before any other crypto
+    /// operations.
+    pub async fn initialize_user_crypto(&self, req: InitUserCryptoRequest) -> Result<()> {
+        Ok(self
+            .0
+            .crypto()
+            .initialize_user_crypto(req)
+            .await
+            .map_err(bitwarden_core::Error::EncryptionSettings)?)
```

Comment:
**suggestion:** If we add bitwarden_core error as transparent #from in this crates error, you should be able to avoid the explicit error conversion and rely on the `thiserror` macro from implementation.

This is the pattern I updated uniffi to use in https://github.com/bitwarden/sdk/pull/1124.

Reply:
I have chosen another path which simply converts all errors to strings. I'll make sure we have a follow-up ticket to investigate the best method for catching errors in JS